app/forms/conversion/create_project_form.rb

```diff
@@ -17,26 +17,25 @@ class NegativeValueError < StandardError; end
   attr_reader :provisional_conversion_date,
     :advisory_board_date
 
-  validates :urn,
-    :incoming_trust_ukprn,
-    :establishment_sharepoint_link,
-    :trust_sharepoint_link,
-    :provisional_conversion_date,
+  validates :provisional_conversion_date,
     :advisory_board_date,
     presence: true
 
-  validates :urn, urn: true
-  validates :incoming_trust_ukprn, ukprn: true
-
   validates :provisional_conversion_date, date_in_the_future: true
   validates :provisional_conversion_date, first_day_of_month: true
```

Comment:
I wonder if these two could be squashed onto one line to match the rest of the validations

`validates :provisional_conversion_date, date_in_the_future: true, first_day_of_month: true`